unsegregated/cp.md

```
@@ -0,0 +1,20 @@
+The following API endpoints have been **deprecated** in PMS 1.3.0-beta.3 and replaced with enhanced alternatives:
+### **MISP License Management**
+| Deprecated Endpoint | Replacement | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)